src/app/docs-pages/feedback-doc/feedback-doc.component.html

```diff
@@ -4,15 +4,25 @@ <h1 class="doc-title">Feedback service
   </h1>
 
   <p class="doc-info">
-    This is the documentation for Feedback service. To use the service, call the "showMessage" method and include the arguments 
-    for message, severity, closeable and duration of the feedback message. The feedback messages appear by default in the top right corner of th UI. 
-  </p>
-  <p class="doc-info">
-    <b>
-      Typical use: When displaying some kind of information after an action. 
-      The feedback messages are categorises in four main topics; success, information, warnings and error messages.
-    </b>
+    Alert messages are messages with information that is important or time sensitive.
+    The goal is to inform and redirect. Should be clear, communicative, descriptive, short, contextual and have the right color.
   </p>
+
+  <div class="doc-info-use">
+    <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
```

Comment:
Bør vi bruke samme emoji her som hdd? eller ser det bedre ut nå? Opp til deg

Reply:
Kan godt gjøre det for å være konsistente!

Reply:
Ja, liker at det blir mer konsistent, så det kan jeg bytte!